packages/main/src/components/Splitter/index.tsx

```diff
@@ -126,7 +124,13 @@ const useStyles = createUseStyles(
       minWidth: '1.5rem !important',
       height: '1.625rem',
       backgroundColor: `${ThemingParameters.sapShell_Background}`,
-      zIndex: '1'
+      zIndex: '1',
+      '&:active': {
+        zIndex: '2'
+      },
+      '&:hover': {
+        backgroundColor: CustomThemingParameters.SplitterGripHoverColor
```

Comment:
Please remove all custom color styles (also the one in line 126) from the `gripButton` and use the default design of the transparent button instead. You can achieve that by adding `design={ButtonDesign.Transparent}` to the `Button` component